tools/ci_build/github/android/mobile_package.required_operators.config

```diff
@@ -24,7 +24,8 @@ ai.onnx;12;Scan
 ai.onnx;13;Scan
 
 # internal ops added by optimizers
-com.microsoft;1;DynamicQuantizeMatMul,FusedConv,FusedGemm,FusedMatMul,MatMulIntegerToFloat,NhwcMaxPool,QLinearAdd,QLinearAveragePool,QLinearConv,QLinearGlobalAveragePool,QLinearMul,QLinearSigmoid
+ai.onnx;1;LayerNormalization
```

Comment:
Would be good to comment that this is incorrectly registered in the ONNX domain even though it's an internal op. 

May be possible to add a registration in the correct domain and mark the old one as deprecated. IIRC that's what we do when internal ops migrate to official onnx ops.